compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.cpp

```diff
@@ -464,6 +465,52 @@ FailureOr<Flow::DispatchRegionOp> Flow::wrapOpInDispatchRegion(
   return newRegionOp;
 }
 
+//===---------------------------------------------------------------------===//
+// UnCSE Clonable ops
+//===---------------------------------------------------------------------===//
+
+namespace {
+struct UnCseClonableOps : public RewritePattern {
+  UnCseClonableOps(MLIRContext *context)
+      : RewritePattern(MatchAnyOpTypeTag(), /*benefit=*/1, context) {}
+
+  LogicalResult matchAndRewrite(Operation *op,
+                                PatternRewriter &rewriter) const override {
+    if (!isa<FunctionOpInterface>(op->getParentOp())) {
```

Comment:
we'll want to remove this check - I'm gearing up to get scf in this layer and this would break that

Reply:
Removed all of this